tests/codemods/test_lazy_logging.py

```diff
@@ -3,7 +3,7 @@
 from core_codemods.lazy_logging import LazyLogging
 
 logging_funcs = {"debug", "info", "warning", "warn", "error", "critical"}
-each_func = pytest.mark.parametrize("func", logging_funcs)
+each_func = pytest.mark.parametrize("func", sorted(logging_funcs))
```

Comment:
@clavedeluna this would be a codemod, although we currently ignore test directories by default. I'd like to update that logic, though, so that the ignored directories are codemod-dependent.

Reply:
sorry which part exactly do you suggest as a codemod? sorting part?